CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## Unreleased
+
+### Breaking Changes
+
+- Enable and rename APM features by default ([#1217](https://github.com/getsentry/sentry-dart/pull/1217)) and ([#1222](https://github.com/getsentry/sentry-dart/pull/1222))
+  - Enable by default
```

Comment:
The `enable` by default section is part of 7.0.0-alpha.2 already, #1217
Only the `rename` section is part of the `Unreleased` section, introduced by #1222